### PR TITLE
Now the database is displayed as an astropy table

### PR DESCRIPTION
**Pull request by Punyaslok**
@Cadair

### CODE REVIEW

File: sunpy/database/database.py [hunk 1]
```diff
@@ -954,6 +955,14 @@ def redo(self, n=1):
         """
         self._command_manager.redo(n)  # pragma: no cover
 
+    def display_entries(self, columns=None, sort=False):
+        print display_entries(self, columns, sort)
```
Comment (Cadair) [inline sunpy/database/database.py]: `print()`

File: sunpy/database/database.py [hunk 2]
```diff
@@ -954,6 +955,14 @@ def redo(self, n=1):
...
+        return
+
+    def show_in_browser(self, columns=None, sort=False):
```
Comment (Cadair) [inline sunpy/database/database.py]: you should expose the `jsviewer` kwarg in the method definition so users can change it.

File: sunpy/database/database.py [hunk 3]
```diff
@@ -954,6 +955,14 @@ def redo(self, n=1):
...
+        display_entries(self, columns, sort).show_in_browser(jsviewer=True)
+        return
```
Comment (Cadair) [inline sunpy/database/database.py]: you don't need the return

File: sunpy/database/tables.py [hunk 4]
```diff
@@ -646,4 +650,23 @@ def display_entries(database_entries, columns, sort=False):
...
+
+    columns : iterable of str
+        The columns that will be displayed in the resulting table. Possible
+        values for the strings are all attributes of :class:`DatabaseEntry`.
+
+    sort : bool (optional)
+        If True, sorts the entries before displaying them.
+
+    """
+    print (create_display_table(database_entries, columns, sort))
```
Comment (Cadair) [inline sunpy/database/tables.py]: style: no space before `(`

File: sunpy/database/database.py [hunk 5]
```diff
@@ -954,6 +955,13 @@ def redo(self, n=1):
         """
         self._command_manager.redo(n)  # pragma: no cover
 
+    def display_entries(self, columns=None, sort=False):
+        print (create_display_table(self, columns, sort))
```
Comment (Cadair) [inline sunpy/database/database.py]: style: no space before `(`

File: sunpy/database/database.py [hunk 6]
```diff
@@ -954,6 +955,13 @@ def redo(self, n=1):
...
+
+    def show_in_browser(self, columns=None, sort=False, **kwargs):
+        jsviewer = kwargs.get('jsviewer', True)
```
Comment (Cadair) [inline sunpy/database/database.py]: just put it in the `def` line:
`def show_in_browser(self, columns=None, sort=False, jsviewer=True)`

File: sunpy/database/tables.py [hunk 7]
```diff
@@ -10,6 +10,7 @@
 import os
 
 from astropy.units import Unit, nm, equivalencies
+import astropy
```
Comment (Cadair) [inline sunpy/database/tables.py]: `import astropy.table` ??

**Comment by Cadair** [comment]
:+1:

File: sunpy/database/tables.py [hunk 8]
```diff
@@ -594,7 +595,7 @@ def entries_from_dir(fitsdir, recursive=False, pattern='*',
             break
 
 
-def display_entries(database_entries, columns, sort=False):
+def create_display_table(database_entries, columns=None, sort=False):
```
Comment (Cadair) [inline sunpy/database/tables.py]: it might be worth making this 'private' i.e. starting it with an `_`

**Comment by Cadair** [comment]
can you add a changelog entry please?

**Comment by wafels** [comment]
+1

**Comment by dpshelio** [comment]
:+1:

**Pull request merged**